pkg/apis/management.cattle.io/v3/authz_types.go

```diff
@@ -326,27 +326,27 @@ type ClusterRoleTemplateBinding struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
```

Comment:
NIT:
While you have this open can you add this comment to meta data. I stole it from upstream
```golang
// Standard object metadata; More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata.
// +optional
metav1.ObjectMeta `json:"metadata,omitempty"`
```